File: src/libs/ThirdParty.VBSuppliers/Properties/ExpediaRapid/SerializableClasses/Search/PropertyAvailablility.vb
Imports Newtonsoft.Json
Namespace ExpediaRapid.SerializableClasses.Search
    Public Class PropertyAvailablility

        <JsonProperty("property_id")>
        Public Property PropertyID As String

        <JsonProperty("rooms")>
        Public Property Rooms As New List(Of SearchResponseRoom)

        <JsonProperty("links")>
        Public Property AvailabilityLinks As New Dictionary(Of String, Link)

        <JsonProperty("score")>
        Public Property Score As Integer

    End Class
End Namespace

File: src/libs/iVectorOne.Suppliers/ExpediaRapid/SerializableClasses/Search/PropertyAvailablility.cs
namespace ThirdParty.CSSuppliers.ExpediaRapid.SerializableClasses.Search
{
    using System.Collections.Generic;
    using Newtonsoft.Json;

    public class PropertyAvailablility
    {

        [JsonProperty("property_id")]
        public string PropertyID { get; set; }

        [JsonProperty("rooms")]
        public List<SearchResponseRoom> Rooms { get; set; } = new List<SearchResponseRoom>();

        [JsonProperty("links")]
        public Dictionary<string, Link> AvailabilityLinks { get; set; } = new Dictionary<string, Link>();

        [JsonProperty("score")]
        public int Score { get; set; }

    }
}